src/PostgresWebsockets/Claims.hs

```diff
@@ -43,6 +45,16 @@ validateClaims requestChannel secret jwtToken time =
       Just _ -> throwError "claim is not string value"
       Nothing -> nonExistingClaim defaultVal name
 
+    claimAsJSONList :: Maybe ByteString -> Text -> Claims -> ExceptT Text IO [ByteString]
+    claimAsJSONList defaultVal name cl = case M.lookup name cl of
+      Just (JSON.String s) -> pure [encodeUtf8 s]
```

Comment:
This does not seem to be what you want. The result of lookup we expect is a `JSON.Array`. This should be easy to troubleshoot once you write more specs for this function.